src/sandstorm/grain.capnp

```diff
@@ -421,9 +421,19 @@ interface SessionContext {
   #
   # The capability is implicitly tied to the user as if via `tieToUser()`.
 
-  request @3 (query :PowerboxQuery) -> (cap :PowerboxCapability);
-  # Searches for capabilities in the user's powerbox matching the given query and displays a
-  # selection UI to the user.  Returns the selected capability.
+  # request is no longer an rpc call on the session. Instead you use the postMessage api to get a
```

Comment:
Let's leave the method defined (since it could make sense at some point), but extend the comment to specify the postMessage approach.